### PR TITLE
Add ShareLaTeX @ TUM to routes

### PR DESCRIPTION
**Pull request by mammuth**

### CODE REVIEW

File: redirect/routes.php [hunk 1]
```diff
@@ -233,15 +233,19 @@ class Route {
...
+            'target'      => 'https://sharelatex.tum.de/',
+        ],
     ];
 
     private $synonyms = [
         'erapra' => 'erapraktikum',
         'ma-sp'  => 'sp-ma',
         'eidi'  =>  'info1',
         'eidi2' =>  'info2',
+        'sharelatex' =>  'latex',
```
Comment (kordianbruck) [inline redirect/routes.php]: This should be other way round `source=>target` - maybe add a comment?

**Comment by jpbernius** [comment]
What about a shorter url like `tex.tum.sexy`? 
`sharelatex.tum.sexy` is longer than `sharelatex.tum.de`.

**Comment by mammuth** [comment]
Well, that's why I added latex.tum.sexy as well. 
It's strange we built the synonymns in a way only one is possible?^^

**Comment by kordianbruck** [comment]
@mammuth you can have multiple synonyms, you are just using them wrong.

File: redirect/routes.php (hunk 1 above)
Comment (mammuth) [reply redirect/routes.php]: Ah, I see how I was using them wrong 🙈 didn't think much about it^^

**Comment by mammuth** [comment]
> @mammuth you can have multiple synonyms, you are just using them wrong 👎
True. Didn't think much about it when I entered it, I just expected synonyms to be like this I guess:
`root_subdomain => ['synonym1', 'synonym2']`
But for querying the synonyms it obviously makes sense to store it the other way around 👍

**Pull request merged**